File: C_module/Assignment 9/Q2.c
#include <stdio.h>

void search(int*,int ,int);
void main() {
    int n,num;
    printf("Size of array :: ");
    scanf("%d",&n);
    int arr[n];
    printf("Enter array Elements :: ");
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }

    printf("\nElements are :: ");
    for(int i=0;i<n;i++){
        printf("%d ",arr[i]);
    }
    printf("\nEnter the number to find :: ");
    scanf("%d",&num);

    search(arr,n,num);
    
} 

void search(int *arr,int n,int num){
    int idx=-1;
    for(int i=0;i<n;i++){
        if(arr[i]==num) {
            idx=i;
            break;
        }
    }
    if(idx== -1) printf("Element not in array");
    else printf("Element found in array at index %d",idx);
}

File: Core_Java_module/Practice/9-16-2025 (Array Practice)/Q2.java
//Search the given number in array.

import java.util.Scanner;
class Q2{
	public static void main(String[] args){
		int n;
		Scanner sc=new Scanner(System.in);
		System.out.print("Enter array size : ");
		n=sc.nextInt();
		int [] arr=new int[n];
		for(int i=0;i<n;i++){
			System.out.print("Enter "+(i+1)+" Element of Array : ");
			arr[i]=sc.nextInt();
		}		
		
		System.out.print("Array Elements are :");
		for(int i=0;i<n;i++){
			System.out.print(arr[i]+" , ");
		}
		System.out.println();

		System.out.print("Enter Number to Search : ");
		int k=sc.nextInt();

		for(int i=0;i<n;i++){
			if(arr[i]==k){
				System.out.print(k+" Found at : "+i+" Index");
				break;
			} 
		}
	} //main ends here
}